src/controllers/__tests__/sessionController.test.ts

```diff
@@ -42,14 +46,21 @@ const mockedUser: User = {
   spid_mobile_phone: "3222222222222" as NonEmptyString,
   wallet_token: mockWalletToken as WalletToken
 };
+const aTokenDurationSecs = 3600;
+const mockGet = jest.fn();
+const mockSmembers = jest.fn();
```

Comment:
what is `mockSmembers` ?

Reply:
is the mock function of redis `smembers` method